lua/entities/gmod_wire_forcer.lua

```diff
@@ -68,7 +68,7 @@ function ENT:Think()
 	}
 
 	if not IsValid(trace.Entity) then return end
-	if not gamemode.Call( "GravGunPunt", self:GetPlayer(), trace.Entity ) then return end
+	if self:GetPlayer():IsValid() and not gamemode.Call( "GravGunPunt", self:GetPlayer(), trace.Entity ) then return end
```

Comment:
We've had this discussion a lot before - that's a requirement on `GetPlayer` that it's not possible to tell from the call site, and using `IsValid(x)` means that the code is still correct even if the contract of `GetPlayer` is changed or is somehow unintentionally violated elsewhere.

My point isn't that "this might break", it's "the preferred way of checking if `x` is valid is `IsValid(x)`, which is consistently a correct way of checking for all values of `x`".